### PR TITLE
ADD stories for layout in mdx as example and regression test

### PR DESCRIPTION
**Pull request by ndelangen**
Issue: https://github.com/storybookjs/storybook/pull/9229#issuecomment-590285405

## What I did
Added a few examples

### CODE REVIEW

File: examples/official-storybook/stories/core/layout.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import { Button } from '@storybook/react/demo';
+
+export default {
+  title: 'Core/Layout MDX',
+};
+
+export const Default = () => <Box>padded by default</Box>;
```
Comment (shilman) [inline examples/official-storybook/stories/core/layout.stories.mdx]: All these functions need to be in a CSF file @ndelangen

**Review by shilman — CHANGES_REQUESTED** [review]
Looks good other than the need for two files

File: examples/official-storybook/stories/core/layout.stories.mdx (hunk 1 above)
Comment (ndelangen) [reply examples/official-storybook/stories/core/layout.stories.mdx]: I deleted all those exports, they aren't needed.

File: examples/official-storybook/stories/core/layout.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { Button } from '@storybook/react/demo';
+
+export default {
```
Comment (shilman) [inline examples/official-storybook/stories/core/layout.stories.mdx]: I think you also need to remove this

File: examples/official-storybook/stories/core/layout.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  title: 'Core/Layout MDX',
+};
+
+<Meta
+  title="Core/Layout MDX"
+  component={Button}
+  id="core-layout-mdx"
+  decorators={[storyFn => <div style={{ backgroundColor: 'yellow' }}>{storyFn()}</div>]}
+  parameters={{ notes: 'component notes' }}
```
Comment (shilman) [inline examples/official-storybook/stories/core/layout.stories.mdx]: didn't we deprecate notes?

**Comment by CodeByAlex** [comment]
@ndelangen when trying this, I’m seeing the docs tab also applying those styles which means that everything in the docs tab gets centered

**Comment by shilman** [comment]
@ndelangen are these showing up in chromatic? not seeing this in the PR badge: "383 tests unchanged"

**Comment by ndelangen** [comment]
They show up, I've already accepted them here:
https://www.chromaticqa.com/build?appId=5a375b97f4b14f0020b0cda3&number=14895

**Review by jonspalmer — APPROVED** [review]
LGTM

**Pull request merged**